### DISCUSSION TITLE
NSW Digital Twin fork codebase

### DISCUSSION THREAD

**Discussion by shu-boom** [TerriaJS/terriajs#5264] [General]
The user interface for NSW Digital Twin platform is different from other Terria platforms. It has features like login and help guide. These features are not available in terriajs or TerriaMap repos. I could not find the code for NSW fork like other projects. Is it private?

**Comment by KeyboardSounds** [comment]
Hi @shu-boom,

The specific NSW DT map codebase is private, and the login feature isn't publicly available, but some of the other UI elements are available in the v8 versions of TerriaMap and terriajs. Check out the `next` branch of both of them and it might have some of the things you're looking for.

**Reply by shu-boom** [reply]
Thanks! I will check them out. I see that v8.0.0 is in alpha. Do you know any release date for stable version?

**Comment by malnajdi** [comment]
Is there a way to learn how to enable a private catalog (login required)
